src/components/molecules/GenreSurveyForm.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+import { useRecoilState } from 'recoil';
+import GenreSurveyImg from '../atoms/GenreSurveyImg';
+import { GenreList, genreListState } from '../../store/atoms/Genre/state';
+import PrimaryButton from '../atoms/PrimaryButton';
+import { GENRE_LIST, Genre } from '../../util/data/GenreDate';
+
+const GenreSurveyForm = () => {
+  const [genreList, setGenreList] = useRecoilState(genreListState);
+
+  const countSelectedGenre = (obj: GenreList) => {
+    return Object.values(obj).filter((value) => value === true).length;
+  };
```

Comment:
그리고 사용처 보니까 어차피 무조건 genreList 에서 true 인 값의 개수를 구하는 로직이던데, 그렇다면 아래처럼 최적화도 가능할듯?

```suggestion
  const countSelectedGenre = useMemo(
    () => Object.values(genreList).filter((value) => value).length,
    [genreList],
  );
```

Reply:
아하 useMemo로 genreList값이 변경될때만  하려고 하는거군여 좋은거같습니다

Reply:
그럴거면 recoil 에서 computed value 로 관리하는게 나을듯?
genreList 자체가 이미 recoil 로 관리되는 값이니까!